pkg/services/internal/bitails/helpers.go

```diff
@@ -12,6 +12,30 @@ import (
 	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/wdk"
 )
 
+// getTxStatus returns:
+//
+//	found=false on 404
+//	found=true  with mined=true/false and height (if mined) on 200
+func (b *Bitails) getTxStatus(ctx context.Context, txid string) (found bool, mined bool, height uint32, err error) {
+	url, err := txStatusURL(b.url, txid)
+	if err != nil {
+		return false, false, 0, fmt.Errorf("build tx status URL: %w", err)
```

Comment:
Since you're using named return values.
```suggestion
		err = fmt.Errorf("build tx status URL: %w", err)
		return
```